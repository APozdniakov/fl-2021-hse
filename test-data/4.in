4
a b c d
1
termin@l 0
1
