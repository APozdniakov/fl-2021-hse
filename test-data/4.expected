T_NUMBER 4 1 0
T_ALPHABET_SYMBOL a 2 0
T_ALPHABET_SYMBOL b 2 2
T_ALPHABET_SYMBOL c 2 4
T_ALPHABET_SYMBOL d 2 6
T_NUMBER 1 3 0
T_ALPHABET_SYMBOL termin 4 0
ERROR in [4; 6], symbol @
